src/store/mod.rs

```diff
@@ -134,6 +134,22 @@ impl Store {
             .map(|_| ())
             .map_err(StoreError::QueryError)
     }
+
+    pub fn remove_account(&self, account_id: u64) -> Result<(), StoreError> {
+        let q = self
+            .db
+            .execute(
+                "DELETE FROM accounts WHERE id = ?",
```

Comment:
If it's simple enough, it would be nice to see a test that tests this function.

Reply:
Yes, to do this we should merge this branch with [`cli testing`](https://github.com/lambdaclass/miden-client/tree/cli-testing)